sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/SimplifyConditionalSuite.scala

```diff
@@ -128,7 +192,7 @@ class SimplifyConditionalSuite extends PlanTest with PredicateHelper {
     // When the conditions in `CaseWhen` are all deterministic, `CaseWhen` can be removed.
     assertEquivalent(
       CaseWhen((isNotNullCond, Subtract(Literal(3), Literal(2))) ::
-        (isNullCond, Literal(1)) ::
+        (isNullCondB, Literal(1)) ::
```

Comment:
You don't need to change the existing tests where possible.

Reply:
Ok, i will try to avoid that.